src/replication.c

```diff
@@ -24,7 +24,8 @@
 #include "istgt_scsi.h"
 #include "assert.h"
 
-int extraWait = 5;
+int io_max_wait_time = 60;
```

Comment:
better to make this uint64 for boundary alignment, as it will be updated in uctl thread, and, used in luworker thread.